packages/fast-components-react-msft/src/subheading/subheading.props.ts

```diff
@@ -0,0 +1,34 @@
+import * as React from "react";
+import { IManagedClasses, ISubheadingClassNameContract  } from "@microsoft/fast-components-class-name-contracts-msft";
+
+export enum SubheadingLevel {
+    _1 = 1,
+    _2 = 2,
+    _3 = 3,
+    _4 = 4,
+    _5 = 5,
+    _6 = 6
+}
+
+export enum SubheadingTag {
+    h1 = "h1",
+    h2 = "h2",
+    h3 = "h3",
+    h4 = "h4",
+    h5 = "h5",
+    h6 = "h6",
+    p = "p"
+}
+
+export interface ISubheadingHandledProps {
+
+    children?: React.ReactNode | React.ReactNode[];

```

Comment:
I probably wouldn't put newlines between each property.  But, we should document the standards we require.  Not sure we have.  So partly asking @janechu @nicholasrice what our standard is?

Reply:
Yep - you should describe what each prop is for - see Heading for details